aws/terminator/data_services.py

```diff
@@ -178,6 +178,27 @@ def terminate(self):
         self.client.delete_db_parameter_group(DBParameterGroupName=self.name)
 
 
+class RdsDbClusterParameterGroup(DbTerminator):
+    @staticmethod
+    def create(credentials):
+        return Terminator._create(credentials, RdsDbClusterParameterGroup, 'rds', lambda client: client.describe_db_cluster_parameter_groups()['DBClusterParameterGroups'])
+
+    @property
+    def id(self):
+        return self.instance['DBClusterParameterGroupArn']
+
+    @property
+    def name(self):
+        return self.instance['DBClusterParameterGroup']
```

Comment:
```suggestion
        return self.instance['DBClusterParameterGroupName']
```

Reply:
@gravesm Ah, thanks for spotting this. hmm, If I reformat line 184 it kinda breaks the consistency in the module. Should I add reformat or is there a way to tell pycodestyle to ignore that particular line by use of a code? Or should I bump the line length in this case.

Reply:
Oh actually withdraw previous statements will re-format I see line 227  is a reformatted line lower down :-) Will reformat and submit shortly. Thanks for your assistance. Much appreciated.